lang/fr/LC_MESSAGES/messages.po

```diff
@@ -1520,6 +1520,12 @@ msgstr "Supprimer"
 msgid "Cancel"
 msgstr "Annuler"
 
+msgid "Invalid Image Format."
+msgstr "Format d'image invalide."
+
+msgid "Images already uploaded."
+msgstr "Images déjà uploadés."
```

Comment:
uploadé -> téléversée

Reply:
I'm not a big fan of `téléversée`
Target people are technical, and know better `uploaded` than `téléversées`, even if there are french
What do you think @cgagnaire & @lpinsivy ?

Reply:
We could also use `chargée` ?